src/buckets/components/context/csvUploaderProvider.tsx

```diff
@@ -68,12 +68,37 @@ export const CsvUploaderProvider: FC<Props> = React.memo(({children}) => {
   const resetUploadState = (): void =>
     setUploadState(RemoteDataState.NotStarted)
 
+  const handleError = (error: Error): void => {
+    setUploadState(RemoteDataState.Error)
+    reportErrorThroughHoneyBadger(error, {
+      name: 'uploadCsv function',
+    })
+    const message = getErrorMessage(error)
+    dispatch(notify(csvUploaderErrorNotification(message)))
+  }
+
+  const parseTagStringValue = (value: string): string => {
+    // Replaces special characters in accordance to Line Protocol standards
+    // https://docs.influxdata.com/influxdb/cloud/reference/syntax/line-protocol/#special-characters
+    return value
+      ? value
+          .replace(/,/g, '\\,')
+          .replace(/ /g, '\\ ')
+          .replace(/=/g, '\\=')
+      : '""'
+  }
+
   const uploadCsv = useCallback(
     (csv: string, bucket: string) => {
       setUploadState(RemoteDataState.Loading)
       setTimeout(() => {
         try {
           const {table} = fromFlux(csv)
+          if (!table.length) {
+            throw new Error(
+              `The CSV could not be parsed. Make sure to include _measurement and _field columns`
```

Comment:
We should make this more actionable. There are several reasons that a CSV parser might fail here, maybe we can be a bit more specific by saying that the CSV must be in annotation format with a link to the docs?

https://docs.influxdata.com/influxdb/cloud/reference/syntax/annotated-csv/